dsp/transform/hilbert_example_test.go

```diff
@@ -0,0 +1,46 @@
+package transform
+
+import (
+	"fmt"
+	"math/cmplx"
+)
+
+func ExampleHilbert_AnalyticSignal() {
+	// Samples is a set of real amplitudes that make up a signal
+	samples := []float64{1, 0, 2, 0, 4, 0, 2, 0}
+
+	// Initialize a Hilbert transform and 'demodulate' to get the analytic signal.
+	// The result is the complex I/Q (In-Phase / Quadrature) demodulation of the input signal.
```

Comment:
These comments should be shorter so that they render nicely in the godoc; the don't get wrapped in the presented code, so we need to do that work here.
```suggestion
	// Initialize a Hilbert transform and 'demodulate' to get the
	// analytic signal.
	// The result is the complex I/Q (In-Phase / Quadrature) demodulation
	// of the input signal.
```